met-web/src/components/layout/Drawer/CustomDrawer.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import AppBar from '@mui/material/AppBar';
+import { Grid, ListItemButton, List, ListItem, ListItemText, Box } from '@mui/material';
+import { useAppSelector } from '../../../hooks';
+import UserService from '../../../services/UserServices';
+import { useNavigate } from 'react-router-dom';
+
+const Routes = [
+    { name: 'Engagements', path: '/' },
```

Comment:
may be we need a better pattern to keep all the routes togother at some place so its easier to find.. we can discuss that later..not as a part of this PR anyway